test.js

```diff
@@ -64,7 +64,6 @@ const names = {
 	],
 	mp3: [
 		'fixture',
-		'fixture-offset1-id3',
```

Comment:
Why was this fixture removed?

Reply:
I removed this fixture because I don't think fixture is a realistic representation of an MP3 file:
1. It is not normal an ID3 header starts at offset 1 or any other arbitrary offset in combination with an MPEG stream.
2. Common tools like Mp3Tag or Foobar2000, and a low level MPEG frame decoder I use, cannot make any sense of this file (neither from the MPEG frames).

I am not sure if it was this file, or another file from the same source, even the extension appeared to be wrong. This kind of fixtures are just causing confusion.